pkg/security/probe/perf_buffer_monitor.go

```diff
@@ -349,14 +349,14 @@ func (pbm *PerfBufferMonitor) CountEvent(eventType model.EventType, timestamp ui
 func (pbm *PerfBufferMonitor) sendEventsAndBytesReadStats(client statsd.ClientInterface) error {
 	var count int64
 	var err error
-	tags := []string{pbm.probe.config.StatsTagsCardinality, "", ""}
+	sharedTags := []string{pbm.probe.config.StatsTagsCardinality, ""}
 
 	for m := range pbm.stats {
-		tags[1] = fmt.Sprintf("map:%s", m)
+		sharedTags[1] = fmt.Sprintf("map:%s", m)
 		for cpu := range pbm.stats[m] {
 			for eventType := range pbm.stats[m][cpu] {
 				evtType := model.EventType(eventType)
-				tags[2] = fmt.Sprintf("event_type:%s", evtType)
+				tags := append(sharedTags, fmt.Sprintf("event_type:%s", evtType))
```

Comment:
I understand why this work here, but I don't find it clear that `append` actually does a copy. Since every tags in this file starts with `pbm.probe.config.StatsTagsCardinality` maybe we could have a function like:
```
func (pbm *PerfBufferMonitor) buildTags(tags ...string) []string {
	return append([]string{pbm.probe.config.StatsTagsCardinality}, tags...)
}
```

Reply:
👍🏻